docs/source/architecture.rst

```diff
@@ -1,15 +1,21 @@
+.. _architecture:
+
 Architecture
 ============
 
 Simplesat is modeled after the Composer library from PHP.
```

Comment:
I think it may be more accurate to say the API is modeled after composer, since the solver itself is not really related to composer.